db/migrate/20170419132148_add_week_to_ticket_purchases.rb

```diff
@@ -0,0 +1,15 @@
+class AddWeekToTicketPurchases < ActiveRecord::Migration
+  class TmpTicketPurchase < ActiveRecord::Base
+    self.table_name = 'ticket_purchases'
+  end
+
+  def change
+    add_column :ticket_purchases, :week, :integer
+
+    TmpTicketPurchase.reset_column_information
+    TmpTicketPurchase.find_each do |purchase|
+      purchase.week = purchase.created_at.strftime('%W')
```

Comment:
It seems we might have ticket_purchase records without a created_at field. Not sure why, but let's take that into consideration.